owncloudApp/src/main/res/layout/preview_audio_fragment.xml

```diff
@@ -14,47 +14,55 @@
 
   You should have received a copy of the GNU General Public License
   along with this program.  If not, see <http://www.gnu.org/licenses/>.
-  
+
 -->
 
-<RelativeLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:id="@+id/top"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:gravity="center"
     tools:context=".ui.preview.PreviewAudioFragment"
     android:filterTouchesWhenObscured="true">
 
-    <FrameLayout
+
+    <androidx.constraintlayout.widget.ConstraintLayout
         android:id="@+id/visual_area"
-        android:layout_width="match_parent"
-        android:layout_height="0dp"
         android:layout_alignParentTop="true"
+        android:layout_centerHorizontal="true"
         android:layout_above="@+id/media_controller"
-        >
+        android:layout_gravity="center"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:padding="40dp">
```

Comment:
I'm not a huge fan of this. Adding a constraint layout just to include an image view may not be the best option. If you need some constraints, please move the whole layout to constraint layout and get rid of the relative one 👍

Reply:
done.